proto/federation/v1alpha1/federation.proto

```diff
@@ -8,6 +8,9 @@ package proto.federation.v1alpha1;
 option go_package = "github.com/cofide/cofide-api-sdk/gen/go/proto/federation/v1alpha1";
 
 message Federation {
-  string from = 1;
-  string to = 2;
+  string from = 1 [deprecated = true];
+  string to = 2 [deprecated = true];
```

Comment:
Should we include these if they're going to be insta-deprecated? Or would that break stuff?